### ISSUE TITLE
QA Ph3_Accessibility - Mobile: Schedule - Filters cannot be collapsed with voice over enabled 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#681]
With voice over enabled, the filter overlay cannot be collapsed.  

Steps: 
1. Enable voice over in device accessibility settings
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Open menu > Select Schedule 
4. Tap to open the filters overlay 
5. Tap double tap outside of the filters overlay to try and close it 
6. Note that filters do not collapse 

Browser/Device: 
Safari iPhone 6S iOS 9

![gio_681](https://cloud.githubusercontent.com/assets/15128124/14476212/468d4804-00bb-11e6-9d3d-1a17b6c88792.PNG)
